Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp

```diff
@@ -2864,7 +2864,7 @@ void JSMessageListener::didReceiveMessage(const IPC::Decoder& decoder)
     auto* globalObject = toJS(m_context);
     JSC::JSLockHolder lock(globalObject->vm());
 
-    auto mutableDecoder = IPC::Decoder::create(decoder.buffer(), decoder.length(), { });
+    auto mutableDecoder = IPC::Decoder::create({ decoder.buffer(), decoder.length() }, { });
```

Comment:
Maybe `decoder` could return the buffer as a span directly.

Reply:
It actually does.  I missed this somehow, and the bots caught the build error.